### ISSUE TITLE
sigmoid_cross_entropy is high

### ISSUE THREAD

**Issue by lc123623** [y0ast/VAE-TensorFlow#8]
I have run your program, but the loss between the input and output is high . That is the sigmiod-cross-entropy-with-logits is high which is about 0.25. I think its not a good model to map the input into the latent feature and reconstruct from the latent feature .

**Comment by y0ast** [comment]
Did you uncover any bugs?

Closing for now, but do let me know if something's wrong.
